src/Features/Core/Portable/Completion/CompletionListProvider.cs

```diff
@@ -11,10 +13,24 @@ namespace Microsoft.CodeAnalysis.Completion
     /// </summary>
     internal abstract class CompletionListProvider
     {
+
+        public async Task ProduceCompletionListAsync(CompletionListContext context)
+        {
+            try
+            {
+                await ProduceCompletionListImplAsync(context).ConfigureAwait(false);
+            }
+            catch (Exception ex)
```

Comment:
Don't we want the non-fatal as a part of a filter so that way we can get the full stack?